import-automation/executor/app/configs.py

```diff
@@ -34,7 +34,7 @@ class ExecutorConfig:
 
     # ID of the Google Cloud project that hosts the executor. The project
     # needs to enable App Engine and Cloud Scheduler.
-    gcp_project_id: str = 'google.com:datcom-data'
+    gcp_project_id: str = 'google.com:datcom-import-automation'
```

Comment:
there is no "google.com:" here. Can you try again with that removed?

Reply:
Done